repo/mfem/package.py

```diff
@@ -4,51 +4,1179 @@
+    #
+    # 2. Verify that the shortened URL for the new version is listed at:
+    #    https://mfem.org/download/
+    #
+    # 3. Use http://getlinkinfo.com or similar to verify that the
+    #    underling download link for the latest version comes has the
+    #    prefix: http://mfem.github.io/releases
+    #
+    # If this quick verification procedure fails, additional discussion
+    # will be required to verify the new version.
+
+    license("BSD-3-Clause")
+
+    # 'develop' is a special version that is always larger (or newer) than any
+    # other version.
+    version("develop", branch="master")
+
+    version(
+        "4.7.0",
+        sha256="5e889493f5f79848f7b2d16afaae307c59880ac2a7ff2315551c60ca54717751",
+        url="https://bit.ly/mfem-4-7",
+        extension="tar.gz",
+    )
+
+    version(
+        "4.6.0",
+        sha256="5fa9465b5bec56bfb777a4d2826fba48d85fbace4aed8b64a2fd4059bf075b15",
+        url="https://bit.ly/mfem-4-6",
+        extension="tar.gz",
+    )
+
+    version(
+        "4.5.2",
+        sha256="7003c908c8265810ff97cb37531521b3aed24959975833a01ea05adfdb36e0f7",
+        url="https://bit.ly/mfem-4-5-2",
+        extension="tar.gz",
+    )
+
+    version(
+        "4.5.0",
+        sha256="4f201bec02fc5460a902596697b6c1deb7b15ac57c71f615b2ab4a8eb65665f7",
+        url="https://bit.ly/mfem-4-5",
+        extension="tar.gz",
+    )
+
+    version(
+        "4.4.0",
+        sha256="37250dbef6e97b16dc9ab50973e8d68bc165bb4afcdaf91b3b72c8972c87deef",
+        url="https://bit.ly/mfem-4-4",
+        extension="tar.gz",
+    )
+
+    version(
+        "4.3.0",
+        sha256="3a495602121b986049286ea0b23512279cdbdfb43c15c42a1511b521051fbe38",
+        url="https://bit.ly/mfem-4-3",
+        extension="tar.gz",
+    )
+
+    version(
+        "4.2.0",
+        sha256="4352a225b55948d2e73a5ee88cece0e88bdbe7ba6726a23d68b2736d3221a86d",
+        url="https://bit.ly/mfem-4-2",
+        extension="tar.gz",
+    )
+
+    version(
+        "4.1.0",
+        sha256="4c83fdcf083f8e2f5b37200a755db843cdb858811e25a8486ad36b2cbec0e11d",
+        url="https://bit.ly/mfem-4-1",
+        extension="tar.gz",
+    )
 
-    depends_on("hiprand", when="+rocm")
-    depends_on("hipsparse", when="+rocm")
+    version(
+        "4.0.0",
+        sha256="df5bdac798ea84a263979f6fbf79de9013e1c55562f95f98644c3edcacfbc727",
+        url="https://bit.ly/mfem-4-0",
+        extension="tar.gz",
+    )
 
+    # Tagged development version used by the laghos package:
+    version(
+        "3.4.1-laghos-v2.0", tag="laghos-v2.0", commit="4cb8d2cbc19aac5528df650b4a41c54158d1d2c2"
+    )
+
+    version(
+        "3.4.0",
+        sha256="4e73e4fe0482636de3c5dc983cd395839a83cb16f6f509bd88b053e8b3858e05",
+        url="https://bit.ly/mfem-3-4",
+        extension="tar.gz",
+    )
+
+    version(
+        "3.3.2",
+        sha256="b70fa3c5080b9ec514fc05f4a04ff74322b99ac4ecd6d99c229f0ed5188fc0ce",
+        url="https://goo.gl/Kd7Jk8",
```

Comment:
I get these are from upstream, but these links will no longer work by August 25th https://developers.googleblog.com/en/google-url-shortener-links-will-no-longer-be-available/